src/App.js

```diff
@@ -1,16 +1,57 @@
 import React from 'react';
+import TodoForm from './components/TodoComponents/TodoForm.js';
+import TodoList from './components/TodoComponents/TodoList.js';
+
 
 class App extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      todos: [],
+      task: "",
+      id: 545484,
+      completed: false
+    };
+  }
+  handleChange = event => {
+    this.setState({
+      [event.target.name]: event.target.value
+    });
+  };
+  handleSubmit = event => {
+    event.preventDefault();
+
+    this.setState({
+      toDoItems: [
+        ...this.state.todos,
```

Comment:
you need to change the key name here to match the key on state that has your todo array (todos)